css/templates/gallery/components/_landing-images.css

```diff
@@ -0,0 +1,581 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /* Override so we can use <figure> */
+ figure {
+  margin: 0;
+}
+
+ .gallery-landing-title {
+  font-weight: 600;
+  line-height: 1.2;
+  width: 100%;
+  height: 100%;
+  max-width: 80vw;
+  padding: 1rem 2vw 4rem;
+  /*
+   * The title font size is linear equation as a function of screen size
+   */
+  font-size: 13vw;
+  pointer-events: none;
+}
+
+@media (--breakpoint-sm) {
+  .gallery-landing-title {
+    font-size: 7vw;
+    padding: 1rem 5vw;
+  }
+}
+
+.amp-carousel-button {
+  background-color: white;
+  box-shadow: -10px 0 20px rgba(0,0,0,0.12);
+  border-radius: 0;
+  width: 50px;
+  height: 89px;
+  cursor: pointer;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+.amp-carousel-button:hover {
+  background-color: yellow;
+}
+
+.amp-carousel-button:focus {
+  outline: none;
+}
+
+.amp-carousel-button-next {
+  background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='18' height='18' fill='%23000' viewBox='0 0 18 18'%3E%3Cpath d='M9 3L7.94 4.06l4.19 4.19H3v1.5h9.13l-4.19 4.19L9 15l6-6z'/%3E%3C/svg%3E");
+  right: 0;
+}
+
+.amp-carousel-button-prev {
+  left: 0;
+  background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='18' height='18' fill='%23000' viewBox='0 0 18 18'%3E%3Cpath d='M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z'/%3E%3C/svg%3E");
+}
+
+@media (max-width: 40.06rem) {
+  .amp-carousel-button {
```

Comment:
This is too generic of a name for the carousel - it should be namespaced like in the instructions
something like `gallery-landing-wall-*`

Just because this is the only carousel on this page does not mean users will not add more carousels, 

If you want to make all the carousel buttons have the same look and feel this approach is valid, how ever i doubt that is what is going on here.

Reply:
On the final version of this template, there's an "inner page" (see here https://ampstart-staging-metalab.firebaseapp.com/templates/gallery/gallery-inner.amp) that uses the same carousel -- would you rather we use the current approach, or should I modify this carousel along with the one on the inner page as you suggested?